docs/operation/operation.md

```diff
@@ -893,6 +893,19 @@ to get the results paginated or getting all of them at the same time.
 curl localhost:9911/routes?offset=200&limit=100
 ```
 
+## Passive health check
+
+Skipper has an option to automatically detect and mitigate faulty backend enpoints, this feature is called
+Passive Health Check(PHC). To enable this feature, you need to provide `-enable-passive-health-check` command line flag to Skipper.
+Given only this flag, Skipper will try to provide PHC functionality with default parameters.
+You can configure those parameters with `-passive-health-check-params` command line flag,
+like this: `-passive-health-check-params=total-requests-threshold=10,timed-out-requests-ratio-threshold=0.1`.
+
+The possible sub-options for `-passive-health-check-params` are:
++ `total-requests-threshold=<int>` - the minimum number of requests per minute per backend pod required to activate PHC for this pod
++ `timed-out-requests-ratio-threshold=<float>` - the minimum ratio of failed requests for every backend pod to make PHC avoid
+sending requests to this pod
```

Comment:
We should describe the logic how it works here. When it decides to disable endpoint and when it enables it back based on the configured and implicit parameters.

Reply:
I have described the logic in this file.

> When it decides to disable endpoint
When the round trip ratio is too high AND amount of requests is big enough

> when it enables it back
When the round trip ration is not too high any more. For skipper to be able to compute this ratio. PHC will still send some minimal amount of request even if endpoint is considered unhealthy.